components/docs-chef-io/content/automate/monitoring.md

```diff
@@ -13,53 +13,76 @@ draft = false
 
-## Overview
+Use the authenticated https endpoint `/status` to monitor your Chef Automate installation.
 
-The `/status` endpoint is an authenticated API endpoint for users who want to monitor their Automate installation by connecting to an http endpoint. Previously, users ran the `chef-automate status` command from the Automate node itself to monitor an Automate installation.
+## Checking the Status Endpoint
 
-## Checking the status endpoint
+The authenticated endpoint `/status` provides status for the overall Chef Automate installation as well as its component services.
+When all Chef Automate component services are up, `/status` returns a response code of 200.
+Otherwise, `/status` returns 500. The status of a service can be `OK`, `UNKNOWN`, or `CRITICAL`.
 
-The authenticated endpoint `/status` provides status for the overall Chef Automate installation as well as its component services. 
-When all Chef Automate component services are up, `/status` returns a response code of 200. Otherwise, `/status` returns 500.
-
-To use `/status`, set up an authentication token that can be used with your monitoring system by following the steps below:
+To use `/status`, set up an authentication token for use with your monitoring system by following the steps below:
 
 1. Generate a token:
 
     ```bash
     chef-automate iam token create --id <token-id> <token-name>
     ```
 
-2. Create a policy that allows your created token to access the `/status` endpoint.
+1. Create a policy that allows your created token to access the `/status` endpoint.
 
     ```bash
     curl -k -H "api-token: <admin-token>" -d '{ "name": "Monitoring", "id": "monitoring", "members": [ "token:<token-id>" ], "statements": [ { "effect": "ALLOW", "actions": [ "system:status:get" ], "projects": [ "*" ] } ] }' -X POST https://automate.example.com/apis/iam/v2/policies?pretty
     ```
 
-3. Test that your token and policy give you access to the `/status` endpoint by running the following command:
+1. Test that your token and policy give you access to the `/status` endpoint by running the following command:
+
     ```bash
     curl -k -H "api-token: <token-id>" https://automate.example.com/api/v0/status?pretty
     ```
-The output appears in the following JSON format:
 
-```json
-    {
-      "ok": true,
-      "services": [
-        {
-          "name": "deployment-service",
+   The output appears in the following JSON format:
+
+   ```json
+       {
+         "ok": true,
+         "services": [
+           {
+             "name": "deployment-service",
+             "status": "OK"
+           },
+           {
+             "name": "backup-gateway",
+             "status": "OK"
+           },
+           {
+             "name": "automate-postgresql",
+             "status": "OK"
+           },
+         ]
+       }
+   ```
+
+   If access is not granted, then the output indicates this:
+
+   ```json
```

Comment:
@yzl I appreciate you providing that perfectly not-perfect example in your comment! Does this incorporate your example as desired?

Reply:
wait, what does it mean `if access is not granted?`

Reply:
Was basing phrasing on [line 38](https://github.com/chef/automate/pull/4259/files#diff-cd26af5d6bb3b04cb8b46e56dce74c3cR38) - "Test that your token and policy give you access to the `/status` endpoint" - but I'm not entirely satisfied with what I put on line 66.

Reply:
@mjingle Not quite. The point I meant to make with the example is that the JSON output for a service will look like

```
{
             "name": "<service-name>",
             "status": "<service-status>"
           },
```

where service-status can be OK (service is up and healthy), CRITICAL (service is responding but not healthy), or UNKNOWN (service is down/unloaded).

Reply:
D'oh! I am following now. Thank you for clarifying @yzl. Let me see if I can change it accordingly.

Reply:
@yzl Changed!